test/integration/next-pages/test/next-build.test.ts

```diff
@@ -105,41 +105,27 @@ test(
     const nextPath = "node_modules/next/dist/bin/next";
 
     console.time("[bun] next build");
-    const bunBuild = Bun.spawn([bunExe(), "--bun", nextPath, "build"], {
+    const bunBuild = Bun.spawn([bunExe(), "--bun", nextPath, "build", "--debug"], {
       cwd: bunDir,
-      stdio: ["ignore", "pipe", "inherit"],
+      stdio: ["ignore", "inherit", "inherit"],
       env: {
         ...bunEnv,
         NODE_ENV: "production",
       },
     });
+    await bunBuild.exited;
+    console.timeEnd("[bun] next build");
+    expect(bunBuild.exitCode).toBe(0);
```

Comment:
Ideally, we'd assert on the `exited` value because exitCode will return null if it segfaulted

Reply:
if it was `null` because of a segfault that would still be caught by `.toBe(0)`